pkg/kubelet/apis/config/types.go

```diff
@@ -88,6 +88,11 @@ type KubeletConfiguration struct {
 	// staticPodPath is the path to the directory containing local (static) pods to
 	// run, or the path to a single static pod file.
 	StaticPodPath string
+	// podLogsDir is a custom root directory path kubelet will use to place pod's log files.
+	// Default: "/var/log/pods/"
+	// Note: it is not recommended to use the temp folder as a log directory as it may cause
```

Comment:
I would probably recommend expanding on this when we open a website PR to document this feature. There should be more details here but I don't think a comment on a types file is the right place to put it.

Reply:
Sure, I'll update website docs once this one is in.